### DISCUSSION TITLE
Feature Request: Support text search with other languages

### DISCUSSION THREAD

**Discussion by yzn-h** [payloadcms/payload#745] [Feature Requests & Ideas]
# Bug Report
when i try to search for Arabic words it does not work

## Expected Behavior

to work like English word when searching for them

## Current Behavior

it is giving me "No Posts found"

## Possible Solution

I don't know 

## Steps to Reproduce

1.make a post with this title for example "السلام"
2. try search for it with the search bar
3.
4.

## Detailed Description

I am using the blog template

**Comment by denolfe** [comment]
Hey @yzn-h , this isn't something we support currently. It looks like it may be possible in the future if we can pass through the  [Mongodb's `$language`](https://www.mongodb.com/docs/manual/reference/operator/query/text) parameter into the search, but it would take some investigation. I'll convert this to a discussion as a feature request, so we can weigh the demand for this.
